using_images/xpaas_images/a_mq.adoc

```diff
@@ -117,11 +117,13 @@ system is discouraged. It is recommended to generate your own KeyStore and
 TrustStore and supply them to the image using the OpenShift secrets mechanism or
 S2I.
 
-== Scalability
+== High-Availability and Scalability
 
-It is possible to run multiple JBoss A-MQ containers on a single OpenShift
-instance at the same time. However, real-time scalability is not supported as
-messages stored on a discarded broker would be lost when downscaling.
+xPaaS A-MQ is supported in two modes:
+
+1) A single A-MQ pod mapped to a Persistent Volume for message persistence. This mode provides message High Availability and guaranteed messaging but does not provide scalability.
+
+2) Multiple A-MQ pods using local message persistence (i.e. no mapped Persistent Volume). This mode provides scalability but does not provide message High Availability or guaranteed messaging.
```

Comment:
The `1)` and `2)` here could just be `.`s (so that they render/match the same as the other topics).

See: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#lists